lib/paper_trail/has_paper_trail.rb

```diff
@@ -115,14 +115,23 @@ def setup_callbacks_from_options(options_on = [])
       end
 
       # Record version before or after "destroy" event
-      def paper_trail_on_destroy(recording_order = 'after')
+      def paper_trail_on_destroy(recording_order = 'before')
         unless %w[after before].include?(recording_order.to_s)
           fail ArgumentError, 'recording order can only be "after" or "before"'
         end
 
-        send "#{recording_order}_destroy",
-             :record_destroy,
-             :if => :save_version?
+        if recording_order.to_s == 'after' and
```

Comment:
I opened https://github.com/airblade/paper_trail/pull/813

I added a spec to test that the warning is actually being issued and it's not passing for some reason. We can discuss on the PR.